docs/theme/included/powerline.rst

```diff
@@ -394,6 +395,15 @@ _______
    The marker string used to indicate the following string is a Red Hat Software
    Collection.
 
+.. attribute:: POWERLINE_TERRAFORM_ENV_MARKER
```

Comment:
Flagging this addition and the color option below as newly added to the Powerline Full docs in this round of updates.